provisioning-telephony/nodejs/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "trunk-creation-tutorial",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Russelle",
+  "license": "MIT",
+  "dependencies": {
+    "express": "^4.17.1",
```

Comment:
It doesn't appear that you're using express in the app. If it's not used, please remove the dependency.